src/pam/pam_oslogin_login.cc

```diff
@@ -279,6 +279,8 @@ PAM_EXTERN int pam_sm_authenticate(pam_handle_t * pamh, int flags,
 
   if (!ParseJsonToKey(response, "status", &status)
       || status != "AUTHENTICATED") {
+    if (ParseJsonToKey(response, "rejectionReason", &status) && !status.empty())
```

Comment:
Can you please add curly braces around this clause?

(I've seen the absence of them bite me one too many times...)